### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1137] [Galaxy Trivia] [answered]
> Do you hate people? ...

**Bonus:** Name the author.

**Comment by peterbaccaro** [comment]
The expression is "**Do you hate people? I don't hate them, I just feel better when they're not around.**" is attributed to **Charles Bukowski**.

**Comment by thuongtruong1009** [comment, marked as answer]
The complete expression is: "Do you hate people? I don't hate them...I just feel better when they're not around."

The author of this quote is Charles Bukowski.
